components/playlist/browser/resources/components/playlistFolder.tsx

```diff
@@ -114,6 +116,57 @@ function EditActionsContainer ({
   )
 }
 
+const StyledEmptyFolderContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 1 0 0;
+  align-self: stretch;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
```

Comment:
Does this scroll if there's a header (i.e. with a back arrow)

Reply:
Yes it does. But I feel like we don't need to, as suggestion area should be fixed to the bottom.